tests/python_tests/test_vlm_pipeline.py

```diff
@@ -269,34 +269,36 @@ def test_sampling(config, cache):
     pipe.generate(prompts[0], image=image, generation_config=config)
 
 
-@pytest.mark.skip(reason="CVS-160580: tests/python_tests/test_vlm_pipeline.py::test_perf_metrics fails")
 @pytest.mark.precommit
 @pytest.mark.nightly
 def test_perf_metrics(cache):
     import numpy as np
+    import datetime
     models_path = get_ov_model("katuni4ka/tiny-random-minicpmv-2_6", cache)
 
     images = [get_image_by_link(image_links[0])]
 
+    start_time = datetime.datetime.now()
     pipe = VLMPipeline(models_path, "CPU")
     result = pipe.generate(prompts[0], images=images, generation_config=GenerationConfig(max_new_tokens=30))
+    generate_time = (datetime.datetime.now() - start_time).total_seconds() * 1000
 
     perf_metrics = result.perf_metrics
 
     assert perf_metrics is not None
 
-    assert 0 < perf_metrics.get_load_time() < 2000
-    assert 0 < perf_metrics.get_num_generated_tokens() < 100
-    assert 0 < perf_metrics.get_num_input_tokens() < 100
-    assert 0 < perf_metrics.get_ttft().mean < 1000
-    assert 0 < perf_metrics.get_tpot().mean < 100
-    assert 0 < perf_metrics.get_ipot().mean < 100
-    assert 0 < perf_metrics.get_throughput().mean < 1000
-    assert 0 < perf_metrics.get_inference_duration().mean < 1000
-    assert 0 < perf_metrics.get_generate_duration().mean < 1000
-    assert 0 < perf_metrics.get_tokenization_duration().mean < 100
-    assert 0 < perf_metrics.get_detokenization_duration().mean < 10
-    assert 0 < perf_metrics.get_prepare_embeddings_duration().mean < 100
+    assert 0 < perf_metrics.get_load_time() < generate_time
+    assert 0 < perf_metrics.get_num_generated_tokens() < generate_time
+    assert 0 < perf_metrics.get_num_input_tokens() < generate_time
+    assert 0 < perf_metrics.get_ttft().mean < generate_time
+    assert 0 < perf_metrics.get_tpot().mean < generate_time
+    assert 0 < perf_metrics.get_ipot().mean < generate_time
+    assert 0 < perf_metrics.get_throughput().mean < generate_time
```

Comment:
Throughput is measured in tokens according to benchmark_genai.cpp. It's not a duration. You should probably compare that with `max_new_tokens/generate_time`

Reply:
I used formula `max_new_tokens/(generate_time - TTFT)`, because we subtract TTFT when calculate throughput, so I did the same here. Otherwise estimated throughput is too low. 
https://github.com/openvinotoolkit/openvino.genai/blob/894e2aadff505cfd03b74560f6c90a8106bdad70/src/cpp/src/perf_metrics.cpp#L107-L110

Reply:
Fixed.